src/Endpoints/ShareOfCheckout.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Alma\API\Endpoints;
+
+use Alma\API\RequestError;
+
+class ShareOfCheckout extends Base
+{
+    const SHARE_OF_CHECKOUT_PATH = '/v1/share-of-checkout/';
+
+    /**
+     * @param array $data
+     *
+     * @throws RequestError
+     */
+    public function share($data)
```

Comment:
@syjust-alma I'm adding a ticket to have a data interface for this endpoint as well in clickup